### PR TITLE
Filter Scenarious.feature was added

### PR DESCRIPTION
**Pull request by PolinaShinkareva**

### CODE REVIEW

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
+And vanacies available with Linux Techology
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:6]: typos in words vanacies and Techology

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Scenario: Drop-down of Competence and Technologies
+When User clicks the fields Competence or Technologies
+Then User sees drop-down menu with available Competencies or Technologies
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:10]: I would recommend to use Scenario outline for this case instead of using "or". It will help to check both of the fields and make tests more specific.

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Scenario: Counter for filters
+When User chooses one or more positions in drop-down menu or unchecks the box
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:13]: I would recommend to be more specific here - how much check boxes tack, what changes you see - maybe using boundary value analysis.

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Then Counter in brackets changes its state (increases or decreases)
+
+Scenario: Display of vacancies by the selected Competence
+When User chooses Security and Compliance competence
+Then User sees list of vacancies ( Information Security Specialist) with parameter Remote/Office
+
+Scenario: Display of vacancies by the selected Technology
+When User choose Linux Technology
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:21]: typo - chooses

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+When User clicks the fields Competence or Technologies
+Then User sees drop-down menu with available Competencies or Technologies
+
+Scenario: Counter for filters
+When User chooses one or more positions in drop-down menu or unchecks the box
+Then Counter in brackets changes its state (increases or decreases)
+
+Scenario: Display of vacancies by the selected Competence
+When User chooses Security and Compliance competence
+Then User sees list of vacancies ( Information Security Specialist) with parameter Remote/Office
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:18]: There is only one vacancy for Security and Compliance so I would recommend mention only that one.

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Scenario: Display of vacancies by the selected Technology
+When User choose Linux Technology
+Then User sees list of vacancies (Senior Linux Engineer, Linux Engineer) with parameters Remote/Office
+
+Scenario: If the vacancy is not found
+When User chooses Security and Compliance Competence and Linux Technology
+Then Use sees text "If you don't see an open position that suits your skills stack and/or professional background but you are interested in working with us � please send your CV to career@quantori.com. We will try to find something special and interesting for you!"
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:26]: typo - User

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (ossoboleva) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:27]: As it was said in the lecture, it is worth using "User _should_ see something" in Then lines for all cases.

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 8]
```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
+And vanacies available with Linux Techology
+
+Scenario: Drop-down of Competence and Technologies
+When User clicks the fields Competence or Technologies
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:9]: Yo should write it the following way:
When user clicks on <name> dropdown
Then <line> should be visible

Examples:
| name | line |
| Competence | Security and Compliance |
|  Technologies |  Linux |

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature (hunk 3 above)
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:13]: To carry out this test you should have more test data in Background as without it this test fails

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature (hunk 5 above)
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:18]: You should first click on "Search" button in order to see that vacancy
same comment for the next tests

File: qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature [hunk 9]
```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature:5]: You should provide specific data what vacancies are available
